.ci/run.sh

```diff
@@ -9,10 +9,27 @@ set -e
 cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
 
+install_docker() {
+	docker_version=${docker_version:-19.03}
+	pkg_name="docker-ce"
+	repo_url="https://download.docker.com/linux/ubuntu"
```

Comment:
before installing docker, could you please check that you are running on ubuntu?

Reply:
ok